### ISSUE TITLE
JetImageLoader should return not just image stream

### ISSUE THREAD

**Issue by artem-zinnatullin** [artem-zinnatullin/jet-image-loader#5]
JetImageLoader should return as the result: image stream, image source (memory cache, storage cache, network), downloading time, etc

**Comment by ghost** [comment]
True. 
Also, status of request: Ok, NetworkIsNotAvaivable, ImageNotFound, UnknownIssue, etc.

**Comment by artem-zinnatullin** [comment]
Yeah, I'll do this, but currently I just have no enough time to work on `JetImageLoader` :(

Thank you for participating in project!
